tests/baselines/reference/jsdocParameterTagSnippetCompletion1.baseline

```diff
@@ -0,0 +1,15640 @@
+// /**
+//  * 
+//  * @p
+//      ^
+// | ----------------------------------------------------------------------
+// | abstract
+// | access
+// | alias
+// | argument
+// | async
+// | augments
+// | author
+// | borrows
+// | callback
+// | class
+// | classdesc
+// | constant
+// | constructor
+// | constructs
+// | copyright
+// | default
+// | deprecated
+// | description
+// | emits
+// | enum
+// | event
+// | example
+// | exports
+// | extends
+// | external
+// | field
+// | file
+// | fileoverview
+// | fires
+// | function
+// | generator
+// | global
+// | hideconstructor
+// | host
+// | ignore
+// | implements
+// | inheritdoc
+// | inner
+// | instance
+// | interface
+// | kind
+// | lends
+// | license
+// | link
+// | linkcode
+// | linkplain
+// | listens
+// | member
+// | memberof
+// | method
+// | mixes
+// | module
+// | name
+// | namespace
+// | overload
+// | override
+// | package
+// | param
+// | private
+// | prop
+// | property
+// | protected
+// | public
+// | readonly
+// | requires
+// | returns
+// | satisfies
+// | see
+// | since
+// | static
+// | summary
+// | template
+// | this
+// | throws
+// | todo
+// | tutorial
+// | type
+// | typedef
+// | var
+// | variation
+// | version
+// | virtual
+// | yields
+// | param {Object} param0 
+// | * @param {Object} [param0.b={ a: 1, c: 3 }] 
+// | * @param {*} param0.b.a 
+// | * @param {*} param0.b.c 
+// | ----------------------------------------------------------------------
+//  */
+// function cc({ b: { a, c } = { a: 1, c: 3 } }) {
+// }
+// /**
+//  * 
+//  * @p
+//      ^
+// | ----------------------------------------------------------------------
+// | abstract
+// | access
+// | alias
+// | argument
+// | async
+// | augments
+// | author
+// | borrows
+// | callback
+// | class
+// | classdesc
+// | constant
+// | constructor
+// | constructs
+// | copyright
+// | default
+// | deprecated
+// | description
+// | emits
+// | enum
+// | event
+// | example
+// | exports
+// | extends
+// | external
+// | field
+// | file
+// | fileoverview
+// | fires
+// | function
+// | generator
+// | global
+// | hideconstructor
+// | host
+// | ignore
+// | implements
+// | inheritdoc
+// | inner
+// | instance
+// | interface
+// | kind
+// | lends
+// | license
+// | link
+// | linkcode
+// | linkplain
+// | listens
+// | member
+// | memberof
+// | method
+// | mixes
+// | module
+// | name
+// | namespace
+// | overload
+// | override
+// | package
+// | param
+// | private
+// | prop
+// | property
+// | protected
+// | public
+// | readonly
+// | requires
+// | returns
+// | satisfies
+// | see
+// | since
+// | static
+// | summary
+// | template
+// | this
+// | throws
+// | todo
+// | tutorial
+// | type
+// | typedef
+// | var
+// | variation
+// | version
+// | virtual
+// | yields
+// | param {Object} param0 
+// | * @param {Object} param0.a 
+// | * @param {*} param0.a.b 
+// | * @param {*} param0.a.c 
+// | * @param {*} param0.d 
```

Comment:
one thing you could do for array binding patterns: generate `[*, *]` for the type, plus *two* tabstops. But that seems like a lot of unnecessarily fancy work.